packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -949,31 +943,37 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         // Only create a SummaryManager if summaries are enabled and we are not the summarizer client
         if (this.runtimeOptions.summaryOptions.generateSummaries === false) {
             this._logger.sendTelemetryEvent({ eventName: "SummariesDisabled" });
-        }
-        if (
-            this.runtimeOptions.summaryOptions.generateSummaries !== false
-            && SummarizerClientElection.clientDetailsPermitElection(this.context.clientDetails)
-        ) {
-            // Create the SummaryManager and mark the initial state
-            this.summaryManager = new SummaryManager(
-                this.summarizerClientElection,
-                this, // IConnectedState
-                this.summaryCollection,
-                this.logger,
-                formRequestSummarizerFn(this.context.loader, this.context.deltaManager),
-                new Throttler(
-                    60 * 1000, // 60 sec delay window
-                    30 * 1000, // 30 sec max delay
-                    // throttling function increases exponentially (0ms, 40ms, 80ms, 160ms, etc)
-                    formExponentialFn({ coefficient: 20, initialDelay: 0 }),
-                ),
-                {
-                    initialDelayMs: this.runtimeOptions.summaryOptions.initialSummarizerDelayMs,
-                },
-                this.runtimeOptions.summaryOptions.summarizerOptions,
-            );
-            this.summaryManager.on("summarizerWarning", this.raiseContainerWarning);
-            this.summaryManager.start();
+        } else {
+            if (this.context.clientDetails.type === summarizerClientType) {
```

Comment:
Basically, only summarizer clients will create a `Summarizer` and all other clients will create a `SummaryManager`.
When a client is elected, the summary manager will spawn a new summarizer client and get its `Summarizer`. Is this understanding correct?
Can we please add this information as comment here. It's non-trivial to follow from code alone.

Reply:
That's correct, will add.